pkg/steps/cluster_claim.go

```diff
@@ -50,7 +53,16 @@ func (s *clusterClaimStep) Description() string                 { return s.wrapp
 func (s *clusterClaimStep) Requires() []api.StepLink            { return s.wrapped.Requires() }
 func (s *clusterClaimStep) Creates() []api.StepLink             { return s.wrapped.Creates() }
 func (s *clusterClaimStep) Objects() []ctrlruntimeclient.Object { return s.wrapped.Objects() }
-func (s *clusterClaimStep) Provides() api.ParameterMap          { return s.wrapped.Provides() }
+func (s *clusterClaimStep) Provides() api.ParameterMap {
+	parameters := s.wrapped.Provides()
+	if parameters == nil {
+		parameters = api.ParameterMap{}
+	}
+	parameters[ClusterClaimCLIParam] = func() (string, error) { //nolint:unparam
```

Comment:
What is `unparam`?

Reply:
At least on my end, the linter was complaining that we never return anything except for `nil` for `error` in this function and was saying we should remove it. But that would break the code because parameters needs the `(string, error)` return type. So I'm just telling the linter to not complain about it.